app/server.py

```diff
@@ -76,6 +76,41 @@ def index():
 
 @app.route('/inventory', methods=[GET])
 def query_prod_info():
+    """
+    Retrieve a list of inventory items
```

Comment:
Seems a little uninformative. Maybe something like "Query a list of inventory items based on given keywords"?
Line 81 is a little bit of confusing as well, "return all details of the all the products" is not the major functionality of this endpoint.

Reply:
hmm, I referred to the professor's notes. But sure I don't mind making it more informative